spec/models/program_spec.rb

```diff
@@ -199,4 +199,21 @@
     end
   end
 
+  describe '#any_event_for_this_date?' do
+    it 'returns type should be boolean' do
+      expect(program.any_event_for_this_date?('').in?([true, false])).to eq(true)
+    end
+
+    it 'returns todays events if date is nil' do
+      date = DateTime.parse(' 00:00').utc
```

Comment:
You are testing that the function is returning events for **today**, then the date needs to be today, and it can go directly into the next statement.